### DISCUSSION TITLE
Feature request: mobile notifications

### DISCUSSION THREAD

**Discussion by SlideeScherz** [community/community#122734] [Mobile] [answered]
<details>
<summary>
ℹ️ Additional Debug Info

Beyond the default behavior for notifications on mobile, this also includes the possibility for web hooks and triggering user created notifications/alerts for: 

- Outages
- Action run summaries
- Failed builds, from other CI platforms 

This is especially useful for workflows that include CI steps outside of GitHub, triggered by actions. 

I also personally would rather receive notifications on mobile rather than email alerts. For those users who prefer email alerts, perhaps a different discussion/issue can be raised for webhooks to integrate into GitHub email notification feature 

</summary>
GitHub for iOS, Version 1.160.0 (129580445)
Device: iPhone 14 Pro (iOS 16.6.1)
</details>

**Comment by HenSquared** [comment, marked as answer]
Thanks for the feedback @SlideeScherz , I've shared this with our team to discuss these ideas. :)
